praw/reddit.py

```diff
@@ -296,19 +297,21 @@ def _prepare_objector(self):
                     'ModmailConversation': models.ModmailConversation,
                     'ModmailMessage': models.ModmailMessage,
                     'Submenu': models.Submenu,
+                    'TrophyList': _TrophyList,
```

Comment:
Can we rename `_TropyList` to `TrophyList` and include it under `models` so that `TrophyList` need not be imported separately?